XMetaL/Display/Summary.css

```diff
@@ -477,13 +479,18 @@ DateLastModified             :before,
 /*SummaryModuleLink            :before, */
 PatientVersionOf             :before,
 ReplacementFor               :before,
```

Comment:
I don't think it's legal to have a comma after the last selector in a list.

Reply:
Did I make that change?  When I started updating the client files all selector lists had commas after the last selector.  Not knowing much about CSS at that time I just went along with it.  Later on I've tried to remove those commas when I thought about it and not create any new rules with it.
Given that XMetaL is enforcing the rules more strictly these days, we might want to create a ticket to correct all those bad boys.

Reply:
Or, if you want me to do it as part of this ticket ...

Reply:
Let's do both. Fix this one here, and create a ticket to find (preferably with a script) and fix the rest.